src/citrine/resources/condition_template.py

```diff
@@ -47,13 +46,13 @@ class ConditionTemplate(AttributeTemplate, Resource['ConditionTemplate'], GEMDCo
     def __init__(self,
                  name: str,
                  bounds: BaseBounds,
-                 uids: Optional[Dict[str, str]] = None,
+                 uids: Optional[Dict[str, str]] = dict(),
```

Comment:
This pattern appears repeatedly.  It is necessary because citrine._serialization.properties actually overloads the uids setter and requires a dictionary rather than an Optional pattern as other fields have.  Addressable, but not the immediate purpose of this PR.